### PR TITLE
Antoniokinghunt/friends_create

### PR DESCRIPTION
**Pull request by 4D-Coder**
## Issue
[#22](https://github.com/LYNK-UP-APP/lynk-up-client/issues/22)
[#26](https://github.com/LYNK-UP-APP/lynk-up-client/issues/26)
## Type of change

    [x] New feature
    [x] Bug Fix
    [x] Refactor

## Description of Change

- Created endpoint for `POST  /users/:user_id/friends`
- Implemented some sad path and testing
- Added conditionals in `setup.py` to analyze environment before choosing local db

    
## Requested feedback

- exceptions and error handling
- serializer construction pattern

## Additional Notes

- updated my desired tickets on projects
- let me know if any questions

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -20,15 +20,17 @@ jobs:
...
             python -m venv .venv
             . .venv/bin/activate
+            /home/circleci/project/.venv/bin/python -m pip install --upgrade pip
             pip install -r lynk_up_server/dependencies.txt
       - run:
           name: "Run tests"
           command: |
             . .venv/bin/activate
-            pytest 
+            cd lynk_up_server
```
Comment (hwryan12) [inline .circleci/config.yml:29]: Everyone will have to check their `DJANGO_SETTINGS_MODULE` to make sure the path is not nested

File: lynk_up_server/lynk_up_server/tests/api_friends_test.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import pytest
+# from rest_framework.test import APIClient
+import vcr
+from .factories import UserFactory
```
Comment (hwryan12) [inline lynk_up_server/lynk_up_server/tests/api_friends_test.py:4]: Nice. I didn't even think to use factories

File: lynk_up_server/pytest.ini [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # pytest.ini
 [pytest]
-DJANGO_SETTINGS_MODULE = lynk_up_server/lynk_up_server.settings
```
Comment (hwryan12) [inline lynk_up_server/pytest.ini:3]: All BE team should make sure their `DJANGO_SETTINGS_MODULE` is good

**Review by hwryan12 — APPROVED** [review]

**Review by ALHelton — APPROVED** [review]
Everyone will also need to make sure that dependencies.txt is installed as there are new packages. I'm curious to learn more about how you're implementing VCR in the testing for this request.

**Pull request merged**